### PR TITLE
Minor: Make apostrophe curly

### PR DESCRIPTION
**Pull request by frankieroberto**

### CODE REVIEW

**Review by 36degrees — APPROVED** [review]
Thanks @frankieroberto  👍🏻 

This just needs a second review from the team and we can get this merged.

**Review by hannalaakso — APPROVED** [review]

**Review by calvin-lau-gds — APPROVED** [review]

**Pull request merged**
